packages/service-frontend/app/deck/[id]/tarot/TaroCardList.tsx

```diff
@@ -73,8 +81,18 @@ function TaroCardList({
   onClickCurrentCard,
 }: Props): JSX.Element {
   const currentCard = cards[currentIndex]
-  const nextCard =
-    currentIndex === cards.length - 1 ? null : cards[currentIndex + 1]
+  const isRightAnimation = currentIndex === cards.length - 1 ? false : true
```

Comment:
마이너한 이슈인데 이거 단순하게 작성할 수도 있을 듯!

```suggestion
  const isRightAnimation = currentIndex !== cards.length - 1
```